ReSharper.FSharp/src/FSharp.Psi.Services/src/Util/FSharpExpressionUtil.fs

```diff
@@ -196,6 +196,29 @@ let tryGetEffectiveParentComputationExpression (expr: IFSharpExpression) =
 
     loop false expr
 
+let isInsideComputationExpressionForCustomOperation (expr: IFSharpExpression) =
```

Comment:
A somewhat corner case example is typing a custom operator before an existing argument:

<img width="635" alt="Screenshot 2023-09-18 at 18 06 27" src="https://github.com/JetBrains/resharper-fsharp/assets/3923587/43aca565-f6eb-4cda-8fed-3e33a20ce005">

Not critical, but can likely be supported with a single addition check.